### PR TITLE
yashim/feat: Added Belarus Location Pages

### PR DESCRIPTION
**Pull request by yashim-deriv**
# Description
Summary
-   Added Minsk, Belarus to the Contact Us page as well as the Careers/Location page

Changes:

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Comment by yashim-deriv** [comment]
Draft as currently pending correct address for the Minsk Office.

File: src/pages/careers/_model/_locations/_contact-details.js [hunk 1]
```diff
@@ -20,6 +23,8 @@ const address_cyprus = ['Level 5, 42 Agias Filaxeos', 'Limassol 3025']
 
 const address_rwanda = ['Level 2 East Wing, Kigali Heights, KG7', 'Avenue, Kigali']
 
+const address_belarus = ['', '', 'Mink, Belarus']
```
Comment (AbdallaMourad) [inline src/pages/careers/_model/_locations/_contact-details.js:26]: Why the first 2 elements are empty?

Comment (AbdallaMourad) [reply src/pages/careers/_model/_locations/_contact-details.js:26]: Is it suppose to be `Minsk, Belarus`

File: src/pages/contact-us/_offices.js [hunk 2]
```diff
@@ -160,7 +165,8 @@ const FullBox = styled(Flex)`
     }
 `
 
-export const Offices = () => {
+// TODO: remove enableBelarus boolean once address is confirmed.
+export const Offices = ({ enableBelarus }) => {
```
Comment (AbdallaMourad) [inline src/pages/contact-us/_offices.js:169]: `enableBelarus` should be in `snake_case`

File: src/pages/careers/_model/_locations/_contact-details.js (hunk 1 above)
Comment (yashim-deriv) [reply src/pages/careers/_model/_locations/_contact-details.js:26]: @AbdallaMourad typo, good catch! will fix

File: src/pages/contact-us/_offices.js [hunk 3]
```diff
@@ -502,9 +508,51 @@ export const Offices = () => {
...
+                        )}
                     </GridLayout>
                 </Content>
             </Container>
         </SectionContainer>
     )
 }
+
+Offices.propTypes = {
+    enableBelarus: PropTypes.bool,
```
Comment (AbdallaMourad) [inline src/pages/contact-us/_offices.js:557]: same here

File: src/pages/contact-us/_offices.js [hunk 4]
```diff
@@ -502,9 +508,51 @@ export const Offices = () => {
                                 </LocalizedLinkText>
                             </BorderBox>
                         </Flex>
+                        {enableBelarus && (
```
Comment (AbdallaMourad) [inline src/pages/contact-us/_offices.js:511]: same here

File: src/pages/contact-us/index.js [hunk 5]
```diff
@@ -50,7 +50,7 @@ const ContactUs = () => {
                 </HeroWrapper>
             </SectionContainer>
             <ContactWays />
-            <Offices />
+            <Offices enableBelarus={true} />
```
Comment (AbdallaMourad) [inline src/pages/contact-us/index.js:53]: here

File: src/pages/careers/_model/_locations/_contact-details.js (hunk 1 above)
Comment (akmal-binary) [reply src/pages/careers/_model/_locations/_contact-details.js:26]: Definitely, Belarus capital is Minsk.

File: src/pages/contact-us/_offices.js (hunk 2 above)
Comment (AbdallaMourad) [reply src/pages/contact-us/_offices.js:169]: as a suggestion you may rename it to `is_belarus_enables` to indicate it holds a boolean value

File: src/pages/careers/locations/_location-layout.js [hunk 6]
```diff
@@ -150,8 +153,8 @@ export const LocationLayout = ({ location, images }) => {
                 img_data={images[location.name]}
             />
             <Container direction="column">
-                <FirstSection padding="12rem 0">
-                    <Header align="center" as="h2" size={'var(--text-size-header-1)'}>
+                <FirstSection padding="8rem 0">
```
Comment (AbdallaMourad) [inline src/pages/careers/locations/_location-layout.js:156]: Can we add `padding="8rem 0"` to `FirstSection` component?

File: src/pages/careers/locations/_location-layout.js [hunk 7]
```diff
@@ -166,12 +169,12 @@ export const LocationLayout = ({ location, images }) => {
...
                 </FirstSection>
             </Container>
-            <SectionContainer padding={location.has_iframe ? '0rem' : '0 0 12rem'}>
+            <SectionContainer padding="0">
                 <Header
                     align="center"
                     as="h2"
-                    size={'var(--text-size-header-1)'}
-                    style={{ marginBottom: '6.4rem' }}
+                    size={'var(--text-size-header-5)'}
```
Comment (AbdallaMourad) [inline src/pages/careers/locations/_location-layout.js:176]:
```suggestion
                    size="var(--text-size-header-5)"
```
I guess it can be used as string. Please check 🙏🏻

File: src/pages/careers/locations/_location-layout.js [hunk 8]
```diff
@@ -166,12 +169,12 @@ export const LocationLayout = ({ location, images }) => {
...
+                    style={{ marginBottom: '4rem' }}
```
Comment (AbdallaMourad) [inline src/pages/careers/locations/_location-layout.js:177]:
```suggestion
                    mb="4rem"
```

File: src/pages/careers/locations/_location-layout.js [hunk 9]
```diff
@@ -212,7 +215,7 @@ export const LocationLayout = ({ location, images }) => {
                     </Flex>
                 </Flex>
             </SectionContainer>
-            <SectionContainer padding="12rem 0">
+            <SectionContainer padding="8rem 0">
```
Comment (AbdallaMourad) [inline src/pages/careers/locations/_location-layout.js:218]: You can remove padding. It will fallback to `8rem 0` by default

File: src/pages/careers/locations/_location-layout.js [hunk 10]
```diff
@@ -15,6 +15,16 @@ const Pin = styled.img`
...
+const StyledBackground = styled(BackgroundImage)`
+    min-width: 100%;
+    height: 80rem;
+    object-fit: contain;
+    @media ${device.tabletL} {
+        height: 65.3rem;
+    }
+`
+
 const StyledContainer = styled(Container)`
```
Comment (AbdallaMourad) [inline src/pages/careers/locations/_location-layout.js:28]: We should use `styled(Flex)` instead of `Container` since we are setting `flex-direction` property

File: src/pages/careers/locations/_location-layout.js [hunk 11]
```diff
@@ -15,6 +15,16 @@ const Pin = styled.img`
     height: 24px;
     margin-right: 13px;
 `
+
+const StyledBackground = styled(BackgroundImage)`
+    min-width: 100%;
```
Comment (AbdallaMourad) [inline src/pages/careers/locations/_location-layout.js:20]: Is there any difference between `min-width` and `width` here?

File: src/pages/careers/locations/_location-layout.js [hunk 12]
```diff
@@ -62,7 +65,7 @@ const Hero = ({ display_name, img_data, description }) => {
                     {`View open positions in ${display_name}`}
```
Comment (AbdallaMourad) [inline src/pages/careers/locations/_location-layout.js:65]: Can you confirm with @mustofa-binary whether we need `localize` for that

File: src/pages/careers/locations/_location-layout.js [hunk 13]
```diff
@@ -150,8 +153,8 @@ export const LocationLayout = ({ location, images }) => {
                 img_data={images[location.name]}
             />
             <Container direction="column">
-                <FirstSection padding="12rem 0">
-                    <Header align="center" as="h2" size={'var(--text-size-header-1)'}>
+                <FirstSection padding="8rem 0">
+                    <Header align="center" as="h2" size={'var(--text-size-header-5)'}>
```
Comment (AbdallaMourad) [inline src/pages/careers/locations/_location-layout.js:157]:
```suggestion
                    <Header align="center" as="h2" size="var(--text-size-header-5)">
```

File: src/pages/careers/locations/_location-layout.js [hunk 14]
```diff
@@ -150,8 +153,8 @@ export const LocationLayout = ({ location, images }) => {
...
                         {`Living in ${display_name}`}
```
Comment (AbdallaMourad) [inline src/pages/careers/locations/_location-layout.js:158]: `localize`

File: src/pages/careers/locations/_location-layout.js (hunk 10 above)
Comment (AbdallaMourad) [reply src/pages/careers/locations/_location-layout.js:28]: If the intention is to use `Container` not `Flex` you can test and remove the unrelated properties 😅

Comment (yashim-deriv) [reply src/pages/careers/locations/_location-layout.js:28]: Apparently this `Container` is not the `Container` we suspected. It is a custom `CareerContainer` component in `/career/_layout-components/_container.js` which is not based off the `Flex` Component :laughing: 

To avoid any confusion I renamed all instances of Container in this file to `CareerContainer`

Comment (AbdallaMourad) [reply src/pages/careers/locations/_location-layout.js:28]: I see 😂 we can move forward with it then 👍🏻

File: src/pages/careers/locations/_location-layout.js (hunk 11 above)
Comment (yashim-deriv) [reply src/pages/careers/locations/_location-layout.js:20]: Reverted this changed in the latest commit. Thanks!

File: src/pages/careers/locations/_location-layout.js [hunk 15]
```diff
@@ -149,31 +153,27 @@ export const LocationLayout = ({ location, images }) => {
...
+                        <Localize translate_text="Living in " />
+                        {display_name}
                     </Header>
                     <Flex tablet_direction="column">
                         <Text size="var(--text-size-sm)">{location.first_p}</Text>
                         <ImageWrapper>
                             <QueryImage
                                 data={images[location.first_img]}
-                                alt={`Living in ${display_name}`}
+                                alt={Localize('Living in ') + display_name}
```
Comment (AbdallaMourad) [inline src/pages/careers/locations/_location-layout.js:167]: Does `Localize` work as a function or should it be `localize`?

Comment (yashim-deriv) [reply src/pages/careers/locations/_location-layout.js]: my bad, fixed

File: src/pages/careers/locations/_location-layout.js [hunk 16]
```diff
@@ -59,10 +62,11 @@ const Hero = ({ display_name, img_data, description }) => {
                     rel="noopener noreferrer"
                     external
                 >
-                    {`View open positions in ${display_name}`}
+                    <Localize translate_text="View open positions in " />
+                    {display_name}
```
Comment (AbdallaMourad) [inline src/pages/careers/locations/_location-layout.js:66]: This should be wrapped up with `Localize`

`<Localize ...props> {display_name} </Localize>
`

File: src/pages/careers/locations/_location-layout.js [hunk 17]
```diff
@@ -149,31 +153,27 @@ export const LocationLayout = ({ location, images }) => {
...
             />
-            <Container direction="column">
-                <FirstSection padding="12rem 0">
-                    <Header align="center" as="h2" size={'var(--text-size-header-1)'}>
-                        {`Living in ${display_name}`}
+            <CareerContainer>
+                <FirstSection>
+                    <Header align="center" as="h2" size="var(--text-size-header-5)">
+                        <Localize translate_text="Living in " />
+                        {display_name}
```
Comment (AbdallaMourad) [inline src/pages/careers/locations/_location-layout.js:160]: same here

File: src/pages/careers/locations/_location-layout.js [hunk 18]
```diff
@@ -149,31 +155,29 @@ export const LocationLayout = ({ location, images }) => {
...
+                            values={{ country: display_name }}
+                        />
                     </Header>
                     <Flex tablet_direction="column">
                         <Text size="var(--text-size-sm)">{location.first_p}</Text>
                         <ImageWrapper>
                             <QueryImage
                                 data={images[location.first_img]}
-                                alt={`Living in ${display_name}`}
+                                alt={localize('Living in ') + display_name}
```
Comment (AbdallaMourad) [inline src/pages/careers/locations/_location-layout.js:171]: `display_name` should be localized as well

**Review by AbdallaMourad — APPROVED** [review]

File: src/pages/home/_trade-types.js [hunk 19]
```diff
@@ -34,7 +34,7 @@ const TradeTypes = () => {
                     </Header>
                     <Text size="2.4rem" mt="1.6rem" mb="5rem">
                         {localize(
-                            'Trade what you like, the way you like it, and on your preferred market.',
+                            'Explore different trade types to trade on your preferred market.',
```
Comment (mustofa-binary) [inline src/pages/home/_trade-types.js:37]: please make a different PR for this change, and based to crowdin

File: src/pages/home/_trade-types.js [hunk 20]
```diff
@@ -60,7 +60,7 @@ const TradeTypes = () => {
 
 const margin = {
     icon: <img src={MarginLogo} alt="margin" width="48" height="49" />,
-    title: <Localize translate_text="Margin Trading" />,
+    title: <Localize translate_text="Margin trading" />,
```
Comment (mustofa-binary) [inline src/pages/home/_trade-types.js:63]: same here

**Review by mustofa-binary — CHANGES_REQUESTED** [review]

File: src/pages/home/_trade-types.js (hunk 19 above)
Comment (yashim-deriv) [reply src/pages/home/_trade-types.js:37]: Ok will revert changes here and separate to another PR

**Review by mustofa-binary — APPROVED** [review]

File: src/pages/home/_trade-types-mobile.js [hunk 21]
```diff
@@ -144,9 +144,7 @@ const TradeTypesMobile = () => {
                     {localize('Trade types')}
                 </Header>
                 <Text padding="0 2rem" align="center" mb="2rem">
-                    {localize(
-                        'Trade what you like, the way you like it, and on your preferred market.',
-                    )}
+                    {localize('Explore different trade types to trade on your preferred market.')}
```
Comment (mustofa-binary) [inline src/pages/home/_trade-types-mobile.js:147]: missed out this one 😅

**Review by mustofa-binary — CHANGES_REQUESTED** [review]

**Review by mustofa-binary — APPROVED** [review]
